example/matrixMulWithMdspan/CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+#
+# Copyright 2023 Erik Zenker, Benjamin Worpitz, Jan Stephan
```

Comment:
Needs to be modiefied

Reply:
ok changed to 2024 and SPDX-License-Identifier: MPL-2.0